r4/v2-to-fhir-custom-mapper-service/service.bal

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import ballerina/http;
+import ballerinax/health.hl7v2commons;
+
+// You can implement custom mapping logic within resource functions for each segment in the HL7v2 message to 
+// convert it to a FHIR resource.
+service /v2tofhir on new http:Listener(9091) {
```

Comment:
Add `custom` part to the context.

Reply:
Fixed with 4ce07f9f38de973d0024c750e49923240e35fc3d